brew/tailscale.rb

```diff
@@ -0,0 +1,87 @@
+# typed: false
+# frozen_string_literal: true
+
+# TODO(mkramlich): generate from template, esp url/tag/rev, version, sha256
+
+# Homebrew formula for Tailscale
+class Tailscale < Formula
+  desc "Easiest, secure, crossplatform way to use WireGuard w/oauth2, 2FA/SSO"
```

Comment:
```suggestion
  desc "The easiest, most secure way to use WireGuard and 2FA"
```
Copied from repo description.